### PR TITLE
[wt] update to 4.6.1

### PR DESCRIPTION
**Pull request by RockinRoel**
- #### What does your PR fix?  
  Updates Wt to 4.6.1

  Setting WTHTTP_CONFIGURATION and CONFIGURATION to empty string as well, because these would otherwise be filled in with an absolute path. Setting them to empty string ensures they won't be used.

  See [issue #9646 on the Wt Redmine](https://redmine.emweb.be/issues/9646)

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  all  
  No

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  Yes

### CODE REVIEW

**Comment by RockinRoel** [comment]
Oops, just noticed PR #22771. I guess that one is a little bit more complete in its removal of absolute paths.

**Comment by autoantwort** [comment]
> Oops, just noticed PR #22771. I guess that one is a little bit more complete in its removal of absolute paths.

You could include the change 
```cmake
# RUNDIR is only used for wtfcgi what we don't build. See https://redmine.webtoolkit.eu/issues/9646
vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/include/Wt/WConfig.h" "#define RUNDIR \"${CURRENT_PACKAGES_DIR}/var/run/wt\"" "")
```
here. 

Thank you for your fast response and your PR here! :)

File: ports/wt/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
   "name": "wt",
-  "version-string": "4.5.0",
-  "port-version": 2,
+  "version-string": "4.6.1",
```
Comment (autoantwort) [inline ports/wt/vcpkg.json:3]:
```suggestion
  "version": "4.6.1",
```

Comment (RockinRoel) [reply ports/wt/vcpkg.json]: Thanks for the suggestion. I changed it.

**Review by JonLiu1993 — APPROVED** [review]

**Comment by JonLiu1993** [comment]
@RockinRoel, Thanks for your pr, I noticed PR #22771 also changed the wt port, If the two prs are merged at the same time, there will be conflicts. I suggest that we wait for pr #22771 to merge, and then update with pr #22771 as the benchmark version. What do you think?

**Comment by RockinRoel** [comment]
@JonLiu1993 Yes, that's fine by me.

**Review by JonLiu1993 — APPROVED** [review]

**Comment by JonLiu1993** [comment]
All features are tested successfully in the following triplet:
- x86-windows
- x64-windows
- x64-windows-static

**Comment by ras0219-msft** [comment]
LGTM, thanks for following up with upstream!

**Pull request merged**